src/test/java/seedu/address/logic/parser/meeting/EditMeetingCommandParserTest.java

```diff
@@ -18,7 +22,6 @@
 import static seedu.address.logic.commands.CommandTestUtil.VALID_DATETIME_TUTORIAL;
 import static seedu.address.logic.commands.CommandTestUtil.VALID_DURATION_LECTURE;
 import static seedu.address.logic.commands.CommandTestUtil.VALID_DURATION_TUTORIAL;
-//import static seedu.address.logic.commands.CommandTestUtil.VALID_MODULE_LECTURE;
 import static seedu.address.logic.commands.CommandTestUtil.VALID_NAME_LECTURE;
```

Comment:
```suggestion
import static seedu.address.logic.commands.CommandTestUtil.VALID_MODULE_INDEX_LECTURE;
import static seedu.address.logic.commands.CommandTestUtil.VALID_NAME_LECTURE;
```
I think the import will go here